todo/src/Helpers/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { tasks } from "../models";
+
+export function numericalOrder() {
+    var sortedTasks = tasks;
+    sortedTasks.sort((task1, task2) => task1.id - task2.id);
+    return sortedTasks;
+}
```

Comment:
A pattern I'm seeing here is that you are assigning `tasks` to a new variable in an attempt to avoid side effects on the underlying `tasks` array.

I think that's a very good thought to have however it's not enough to just assign it to another variable as arrays are reference types and therefore when you reassign you are only assigning the reference in memory to a new variable, a new array is not created.

If you really wanted to avoid side effects you would have to figure out a way to clone the array and make a new one yourself, or learn about immutable ways of doing what you are trying to do (immutable as in the sort operation does not actually change the array it is operating on but rather creating a new array with the result of the sort operation.)